### PR TITLE
Add experts map section to compiler team page

### PR DESCRIPTION
**Pull request by spastorino**
@nikomatsakis @davidtwco added a section to link the experts map.

### CODE REVIEW

File: src/compiler-team.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ home. In any case, you can find people in one of three places at the moment:
 - The `t-compiler` stream on [the Zulip instance](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler)
 - The `compiler` channel on the [rust-lang discord](https://discord.gg/rust-lang)
 
+## Expert map
+
+We maintain a [list of areas of the compiler and who on the team knows a
+bit about that particular area](https://github.com/rust-lang/compiler-team/blob/master/experts/MAP.md).
+This may be useful when you're working on some specific thing and may
+need somebody to help you.
```
Comment (davidtwco) [inline src/compiler-team.md]:
```suggestion
need somebody to help you or when looking for reviewers for a PR.
```

File: src/compiler-team.md [hunk 2]
```diff
@@ -17,6 +17,13 @@ home. In any case, you can find people in one of three places at the moment:
 - The `t-compiler` stream on [the Zulip instance](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler)
 - The `compiler` channel on the [rust-lang discord](https://discord.gg/rust-lang)
 
+## Expert map
+
+We maintain a [list of areas of the compiler and who on the team knows a
+bit about that particular area](https://github.com/rust-lang/compiler-team/blob/master/experts/MAP.md).
+This may be useful when you're working on some specific thing and may
```
Comment (Centril) [inline src/compiler-team.md]:
```suggestion
This is useful when you're working on some specific thing and
```

File: src/compiler-team.md [hunk 3]
```diff
@@ -17,6 +17,13 @@ home. In any case, you can find people in one of three places at the moment:
 - The `t-compiler` stream on [the Zulip instance](https://rust-lang.zulipchat.com/#narrow/stream/131828-t-compiler)
 - The `compiler` channel on the [rust-lang discord](https://discord.gg/rust-lang)
 
+## Expert map
+
+We maintain a [list of areas of the compiler and who on the team knows a
+bit about that particular area](https://github.com/rust-lang/compiler-team/blob/master/experts/MAP.md).
```
Comment (mark-i-m) [inline src/compiler-team.md]: This link is a 404 for me.

Comment (davidtwco) [reply src/compiler-team.md]: It is intended to be until a compiler-team PR lands.

Comment (mark-i-m) [reply src/compiler-team.md]: For posterity: https://github.com/rust-lang/compiler-team/pull/40

Comment (nikomatsakis) [reply src/compiler-team.md]: I would modify this paragraph in the same way that I suggested on rust-lang/compiler-team#40, I think -- i..e., to give a better idea of how the list is meant to be used, and invite people to ask questions.

**Pull request merged**